src/style.css

```diff
@@ -1,3 +1,146 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  font-family: 'Poppins', sans-serif;
+  padding: 10px;
+}
+
+.container {
+  margin: 20px auto;
+  max-width: 800px;
+  padding: 20px 30px;
+  box-shadow: 3px 2px 10px 7px rgba(27, 0, 0, 0.17);
+  border-radius: 5px;
+  animation: fade-in 1s ease-in;
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.top {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: rgb(36, 3, 51);
+}
+
+.to-do-add {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.form {
+  width: 98%;
+  margin-block: 20px;
+  animation: slide-up 0.5s ease-in;
+}
+
+@keyframes slide-up {
+  0% {
+    transform: translateY(10px);
+    opacity: 0;
+  }
+
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+input[type="text"] {
+  line-height: 2;
+  padding: 5px 10px;
+  font-style: italic;
+  width: 98%;
+  border: 1px solid gray;
+  border-radius: 5px;
+  font-family: inherit;
+}
+
+i {
+  cursor: pointer;
+  font-size: 1.2rem;
+  transition: color 0.3s ease-in;
+}
+
+i:hover {
+  color: red;
+}
+
+.to-do-clear {
+  background-color: gainsboro;
+  width: 100%;
+  margin: 5px auto;
+  display: block;
+  padding: 10px 20px;
+  border: none;
+  font-size: 1.3rem;
+  font-family: inherit;
+  cursor: pointer;
+  transition: all 0.3s ease-in;
+}
+
+.to-do-clear:hover {
+  background-color: black;
+  color: #ffff;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.each-todo {
+  display: flex;
+  justify-content: flex-start;
+  gap: 20px;
+  position: relative;
+  width: 100%;
+  margin-block: 10px;
+  border-bottom: 1px solid gray;
+  border-top: 1px solid gray;
+  padding-block: 5px;
+  animation: fade-in 0.5s ease-in;
+}
```

Comment:
- [ ] You did well 👏 . But as stated by the previous reviewer, follow the project requirement and make sure that the list is a clone of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).
- I can see added lines in your UI compared to the simple yet well to do design given

## Your work
![image](https://github.com/kayjeee/webpack-todolist/assets/50990925/3b2978c1-32db-4f1d-b1a0-107ced9e2a20)
 
 ## Design given:
![image](https://github.com/kayjeee/webpack-todolist/assets/50990925/2cb49371-47cf-42f4-b7db-37145156f97c)

Reply:
Hi  @Shedrack-Sunday 

Thank you for the good feedback
I removed the lines as requested